### PR TITLE
RYA-402 Create Kafka reusable test code project.

### PR DESCRIPTION
**Pull request by kchilton2**
[Brief Description of what changed]

[Description of what tests were written]

### Links
[Jira](https://issues.apache.org/jira/browse/RYA-NUMBER)

### Checklist
- [ ] Code Review
- [ ] Squash Commits

#### People To Reivew
[Add those who should review this]

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/524/

File: test/kafka/pom.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                    <groupId>org.slf4j</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    
+        <!-- Testing dependencies. -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```
Comment (jdasch) [inline test/kafka/pom.xml]: Not sure if you still need this version override.  If you do, I'll take a wag at it when I tackle RYA-366.

Comment (kchilton2) [reply test/kafka/pom.xml]: Removed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/525/

**Comment by kchilton2** [comment]
Fixed all of those problems.

**Comment by kchilton2** [comment]
The integration tests are passing now.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/526/

**Pull request closed without merging**